README.md

```diff
@@ -13,37 +24,34 @@ Libernet is open source web app for tunneling internet using SSH, V2Ray, Trojan,
 - Shadowsocks
 - go-tun2socks
 - badvpn-tun2socks (legacy)
-- dnsmasq
+- dnsmasq-full
 - https-dns-proxy
 - php7
 - php7-cgi
 - php7-mod-session
 - php7-mod-json
 
-## Working Features:
-- SSH with proxy
-- SSH-SSL
-- V2Ray VMess
-- V2Ray VLESS
-- V2Ray Trojan
-- Trojan
-- Shadowsocks
+</p>
+</details>
 
-## Installation
+### Installation
 - If you don't have git on OpenWRT, please install first: ```opkg update && opkg install git```
 - Prepare installation directory: ```mkdir -p ~/Downloads && cd ~/Downloads```
 - Clone this repository: ```git clone git://github.com/lutfailham96/libernet.git```
 - Run installation script: ```cd libernet && bash install.sh```
-- Open Libernet on http://router-ip/libernet
+- Open Libernet on http://openwrt.lan/libernet
```

Comment:
Can you just leave this to original, because some people don't understand & can't visit these local domain